Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -670,21 +698,26 @@ public enum AccessTypes
     {
         /// <summary> ID based door access. </summary>
         Id,
+
         /// <summary>
         /// Allows everyone to open doors, except external which airlocks are still handled with ID's
         /// </summary>
         AllowAllIdExternal,
+
         /// <summary>
         /// Allows everyone to open doors, except external airlocks which are never allowed, even if the user has
         /// ID access.
         /// </summary>
         AllowAllNoExternal,
+
         /// <summary> Allows everyone to open all doors. </summary>
         AllowAll
```

Comment:
```suggestion
        AllowAll,
```

Reply:
As much as I agree with you, out of scope